services/users/services.go

```diff
@@ -0,0 +1,75 @@
+/*This file contains the required businees logic implementation. Internally it is using the
+  CRUD functinality provided in the stores package*/
+
+package users
+
+import (
+	"errors"
+
+	"github.com/arohanzst/user-curd/models"
+	"github.com/arohanzst/user-curd/services"
+	"github.com/arohanzst/user-curd/stores"
+)
+
+type User struct {
+	u stores.User
+}
+
+func New(u stores.User) services.User {
+	return &User{u}
+}
+
+//Creates a New User
+func (st *User) Create(value models.User) (models.User, error) {
+	user, err := st.u.Create(value)
+
+	if err != nil {
+		return models.User{}, errors.New("Given user could not be created.")
```

Comment:
just return the error coming from the store layer
if err != nil {
		return nil,err
		}